test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```diff
@@ -130,6 +130,11 @@ spec:
       openAPIV3Schema:
         type: boolean
         default: false
+  - name: debugLogging
+    schema:
+      openAPIV3Schema:
+        type: boolean
+        default: false
```

Comment:
❓ : why not make it an optional int and in our templates above we could set it to `4`? This way we could also increase/decrease the `v` value :-)

But also okay with using a bool.

Reply:
Let's use a bool for now for simplicity.